runtime/rasdump/javadump.cpp

```diff
@@ -4603,10 +4648,21 @@ JavaCoreDumpWriter::writeThread(J9VMThread* vmThread, J9PlatformThread *nativeTh
 #else /* defined(J9ZOS390) || defined(J9ZTPF) */
 		_OutputStream.writeCharacters("3XMTHREADINFO3           No native callstack available for this thread\n");
 #endif /* defined(J9ZOS390) || defined(J9ZTPF) */
-		_OutputStream.writeCharacters("NULL\n");
 	}
+#if JAVA_SPEC_VERSION >= 21
+	if ((NULL != vmThread) && (NULL != vmThread->threadObject)) {
+		_OutputStream.writeCharacters("3XMTHREADINFO4           Type:");
+		if (IS_JAVA_LANG_VIRTUALTHREAD(vmThread, vmThread->threadObject)) {
+			_OutputStream.writeCharacters("Virtual, java/lang/CarrierThread:");
```

Comment:
Space after `:`, please (on lines 4654 and 4656).

Reply:
In the javacore output, currently all thread related data do not have a space between `:` and the pointer/value, should all be updated?

Reply:
It's not consistent, for example there's a space written after the colon in `JavaCoreDumpWriter::writeThreadTime()`. I think any additions should be consistent. If you can point to a place where similar information is written without the colon, then omit the space, otherwise there's normally a space (or two in English) after a colon.

Reply:
All of the numeric fields in `JavaCoreDumpWriter::writeThread` don't have spaces, which is the format that I am trying to follow, will still add the space for any non-numeric fields